src/eventer/eventer_epoll_impl.c

```diff
@@ -448,8 +453,9 @@ static void eventer_epoll_impl_trigger(eventer_t e, int mask) {
       }
       master_fds[fd].e = NULL;
     }
-    eventer_free(e);
+    eventer_deref(e);
   }
+  eventer_deref(e);
```

Comment:
Won't the fall-through from line 456 cause this to deref twice?  This is in a few other places too.

Reply:
Correct. Intentional... The first one is the deref b/c the event is no longer used.  The second one is the deref matching the ref we did in the beginning of the function.